content-src/components/LinkMenu/LinkMenu.js

```diff
@@ -12,13 +12,15 @@ const LinkMenu = React.createClass({
     };
   },
   userEvent(event) {
-    const {page, source, index, dispatch} = this.props;
+    const {page, source, index, dispatch, site} = this.props;
     if (page && source) {
       let payload = {
         event,
         page: page,
         source: source,
-        action_position: index
+        action_position: index,
+        url: site.recommended ? site.url : null,
+        recommender_type: site.recommended ? site.recommender_type : null
```

Comment:
Yeah, definitely double check before listening to me (good advice in general).
I did see we specify `propTypes` below which defines a `site` shape object with a required `url` string (lines 112-116 below), so maybe that's enough to guarantee we'll always have that object.